etc/profile-a-l/ledger-live-desktop.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for Ledger Live
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ledger-live-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Ledger Live
+
+mkdir ${HOME}/.config/Ledger Live
+whitelist ${HOME}/.config/Ledger Live
+whitelist ${DOWNLOADS}
+whitelist /opt/ledger-live
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-runuser-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nosound
+novideo
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+tracelog
+
+disable-mnt
+private-cache
+# enabling private-dev blocks USB hardware wallets
+# private-dev
+private-etc @network,@tls-ca,@x11
```

Comment:
I' not running firejail from master so my actual profile uses:

```
private-etc alternatives,ca-certificates,crypto-policies,host.conf,nsswitch.conf,pki,resolv.conf,rpc,selinux,ssl
```

Networking breaks without these options.

Reply:
Good you referenced your private-etc in 0.9.72-format. It translates as private-etc @network,@tls-ca,@x11,host.conf,rpc so you'll need to append `,host.conf,rpc` here for master.